ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-master.yaml

```diff
@@ -53,7 +57,11 @@ images:
   to: sriov-network-webhook
 operator:
   bundles:
-  - dockerfile_path: bundleci.Dockerfile
+  - as: sriov-network-operator-bundle
+    base_index: operator-index
+    context_dir: config
```

Comment:
`bundleci.Dockerfile` is on the repository [root](https://github.com/openshift/sriov-network-operator/blob/master/bundleci.Dockerfile)

```suggestion
```